prepare-build.sh

```diff
@@ -1,15 +0,0 @@
-#!/bin/bash
-
-set -e
-
-cd build
-mv static resources
-mv favicon.png resources/
-mv media/* resources/media
-rm -r media
-
-if [ "$(uname)" = "Darwin" ]; then # MacOSX case
-    sed -i '' -e 's/static\///g; s/favicon\.png/resources\/favicon\.png/' index.html
-else # Linux case
-    sed -i -e 's/static\///g; s/favicon\.png/resources\/favicon\.png/' index.html
-fi
```

Comment:
these are two replaces
1. to transform paths like `/resources/static/anything.css/` into `/resource/anything.css` (again, `publicPath` only sets a prefix to all paths, but the target dir was still `static`, hence incorrect paths)
2. to adjust favicon path after moving it a few lines above

2 is no longer needed, since the favicon is initially in the right place
1 is no longer needed, since all paths are correctly created during the bulid

there was also a bug: 1 replaced paths in index.html, but not in styles or scripts, while there also were paths that required similar change